poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFSheet.java

```diff
@@ -191,8 +192,13 @@ public void removeRow(Row row) {
      * @return Row representing the rownumber or null if its not defined on the sheet
      */
     @Override
-    public SXSSFRow getRow(int rownum) {
-        return _rows.get(rownum);
+    public Row getRow(int rownum) {
+    	Row row = _rows.get(rownum);
+    	// jlolling: allow reading all the content
```

Comment:
In the ASF, we believe in shared ownership of the code. Please do not annotate code with your name.